include/rocksdb/utilities/stackable_db.h

```diff
@@ -371,6 +371,10 @@ class StackableDB : public DB {
     return db_->GetSortedWalFiles(files);
   }
 
+  virtual Status GetCurrentWalFile(LogFile** current_log_file) override {
```

Comment:
Similar functions would have the argument as `std::unique_ptr<LogFile>*` instead, so that the ownership is clear.